contracts/LockedTokens.cdc

```diff
@@ -285,8 +285,12 @@ pub contract LockedTokens {
 
             self.address = lockedAddress
             self.tokenManager = tokenManager
-            self.nodeStakerProxy = nil
-            self.nodeDelegatorProxy = nil
+
+            // Create a new staker proxy that can be accessed in transactions
+            self.nodeStakerProxy = LockedNodeStakerProxy(tokenManager: self.tokenManager)
+
+            // create a new delegator proxy that can be accessed in transactions
+            self.nodeDelegatorProxy = LockedNodeDelegatorProxy(tokenManager: self.tokenManager)
```

Comment:
So in the case of someone calling `createNodeStaker` after this upgrade, this will just get over-written, right?

Reply:
yeah, but I can actually just remove the lines in `createNodeStaker` because they aren't needed anymore